lightning_client.go

```diff
@@ -1845,12 +1848,18 @@ func (s *lightningClient) PendingChannels(ctx context.Context) (*PendingChannels
 			return nil, err
 		}
 
+		hash, err := chainhash.NewHashFromStr(waiting.ClosingTxid)
```

Comment:
Are we sure this is always set? If this was only added in 0.14.2 then we'd need to bump the `minimalCompatibleVersion` to make sure we never run into an error here.

Reply:
Fixed.